docs/multi-stage-query/reference.md

```diff
@@ -90,6 +93,66 @@ can precede the column list: `EXTEND (timestamp VARCHAR...)`.
 
 For more information, see [Read external data with EXTERN](concepts.md#read-external-data-with-extern).
 
+#### `EXTERN` to export to a destination
+
+`EXTERN` can be used as a destination, which will export the data to the specified location and format. EXTERN when
```

Comment:
```suggestion
`EXTERN` can be used to specify a destination, where the data needs to be exported.  This variation of EXTERN requires one argument - (What the argument is)
```
minor nit: Also, from the docs, the format is specified via `AS` clause and not extern, therefore I have omitted that from the suggestion